lib/identity-api-client/industries.rb

```diff
@@ -0,0 +1,34 @@
+module IdentityApiClient
+  class Industries < Base
+    def list
+      resp = client.get_request("/api/industries?api_token=#{client.connection.configuration.options[:api_token]}")
+      if resp.status == 200
+        return resp.body
+      else
+        return false
+      end
+    end
+
+    def approved
+      resp = client.get_request("/api/industries/approved?api_token=#{client.connection.configuration.options[:api_token]}")
+      if resp.status == 200
+        return resp.body
+      else
+        return false
+      end
+    end
+
+    def create(industry)
+      params = {
+        'api_token' => client.connection.configuration.options[:api_token],
+        'industry' => industry
+      }
+      resp = client.post_request('/api/industries', params)
+      if resp.status < 400
```

Comment:
I guess we can always be confident we're getting something sensible back in the body? IE not risking a 204 with no content or similar?

Reply:
Yeah, good shout. Looking at the Identity CREATE APIs for Industries, Workplaces, Professions, looks like they'll all either return HTTP 200 and render the created object, or they'll throw an error and return an HTTP 500.

I copy/pasted this template for the create calls from other exiting code in this `identity-api-client` lib, so I _assume_ somewhere in ID there are some CREATE APIs which return non-200 codes on success in some circumstances, but for these ones I think we could just change to explicitly check for a 200 code.